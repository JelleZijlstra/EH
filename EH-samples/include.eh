#!/usr/bin/ehi
# Testing the include: function
include: 'end.eh'
echo 'We just included a file'
maha = new Foo
printvar: maha
printvar: baz
