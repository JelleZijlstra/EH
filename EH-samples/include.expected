0
1
3
42
42
42
We just included a file
@object <Foo> [
bar <public,non-static,non-constant>: @function <user>: n
finalize <public,non-static,non-constant>: @function <library>: 
initialize <public,non-static,non-constant>: @function <library>: 
new <public,non-static,non-constant>: @function <library>: 
toString <public,non-static,non-constant>: @function <library>: 
]
@function <user>: n
